### DISCUSSION TITLE
Box shadow not working on ios

### DISCUSSION THREAD

**Discussion by mimoudix** [tailwindlabs/tailwindcss#12023] [Help] [answered]
Since Iphone ios tends to mess box shadow on inputs, it is necessary to add "-webkit-appearance: none;" manually in css.
Does TailwindCSS comes with prebuilt css to fix this issue ?

**Comment by adamwathan** [comment, marked as answer]
Hey! You can use the `appearance-none` class to add those styles to any element as needed 👍🏻

You can also add whatever custom CSS you want to your CSS file if there are styles you want to apply globally.

**Reply by mimoudix** [reply]
Thank you !
